README.md

```diff
@@ -126,8 +129,9 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
 - [Less Loader](https://github.com/webpack/less-loader): Less loader module for Webpack. -- *Maintainer*: `Webpack Team` [![Github][githubicon]](https://github.com/webpack)
 - [Stylus Loader](https://github.com/shama/stylus-loader): A stylus loader for webpack. -- *Maintainer*: `Kyle Robinson Young` [![Github][githubicon]](https://github.com/shama) [![Twitter][twittericon]](https://twitter.com/shamakry)
 - [Isomorphic Style Loader](https://github.com/kriasoft/isomorphic-style-loader): Isomorphic CSS style loader for Webpack. -- *Maintainer*: `Kriasoft Team` [![Github][githubicon]](https://github.com/kriasoft) [![Twitter][twittericon]](https://twitter.com/kriasoft)
-- [Mini css extract plugin](https://github.com/webpack-contrib/mini-css-extract-plugin): 
+- [Mini css extract plugin](https://github.com/webpack-contrib/mini-css-extract-plugin):
```

Comment:
Here loader list, not plugin

Reply:
Actually, this is not my plugin, and it's already there...But I can move it to plugins.

Reply:
@zaaack thanks :+1: